kernel/src/process/posix_thread/thread_local.rs

```diff
@@ -127,18 +136,17 @@ impl ThreadLocal {
     }
 
     pub fn borrow_file_table(&self) -> FileTableRef {
-        FileTableRef(self.file_table.borrow())
+        ThreadLocalOptionRef(self.file_table.borrow())
     }
 
     pub fn borrow_file_table_mut(&self) -> FileTableRefMut {
```

Comment:
I think the visibility can be restricted now?

Same for `borrow_fs_mut`.

Same for `ThreadLocalOptionRefMut::replace`.

Reply:
`borrow_fs_mut`, `ThreadLocalOptionRefMut::replace`, and `borrow_ns_proxy_mut` are now restricted to `crate::process`.

`borrow_file_table_mut` cannot be restricted because it is still widely used throughout the syscall module.